internal/runbits/dependencies/changesummary.go

```diff
@@ -130,8 +127,9 @@ func OutputChangeSummary(out output.Outputer, newBuildPlan *buildplan.BuildPlan,
 			item = fmt.Sprintf("[ACTIONABLE]%s@%s[/RESET] → %s (%s)", oldVersion.Name, oldVersion.Version, item, locale.Tl("updated", "updated"))
 		}
 
-		out.Notice(fmt.Sprintf("  [DISABLED]%s[/RESET] %s", prefix, item))
+		items[i] = item
 	}
 
+	out.Notice(renderers.NewBulletList("  ", renderers.BulletTree, items))
```

Comment:
Fwiw, with regards to dropping the magical output handing, this could simply become:

```
	out.Notice(renderers.NewBulletList("  ", renderers.BulletTree, items).String())
```